api-report/tree.api.md

```diff
@@ -45,23 +45,23 @@ export function buildForest(): IEditableForest;
 export type ChangeFromChangeRebaser<TChangeRebaser extends ChangeRebaser<any, any, any>> = TChangeRebaser extends ChangeRebaser<infer TChange, any, any> ? TChange : never;
 
 // @public
-export interface ChangeRebaser<TChange, TFinalChange, TChangeSet> {
-    compose(...changes: TChangeSet[]): TChangeSet;
+export interface ChangeRebaser<TChange, TFinalChange, TChangeset> {
+    compose(...changes: TChangeset[]): TChangeset;
```

Comment:
@yann-achard-MS - The capitalization of 'Changeset' come up in another PR, so I did a quick sweep of /tree/.

The prevailing convention is a lower-case 's' (changeset is one word, like hashtable.)